src/resources/2019/CampAway.ts

```diff
@@ -0,0 +1,90 @@
+import {
+  Effect,
+  daycount,
+  myPath,
+  mySign,
+  retrieveItem,
+  use,
+  visitUrl,
+} from "kolmafia";
+import { signIdToName } from "../../ascend";
+import { get, withChoice } from "../../property";
+import { $item } from "../../template-string";
+import { random } from "../../utils";
+
+/**
+ * Determines whether we `have` the campsite
+ *
+ * @returns Whether we `have` the campsite
+ */
+export function have() {
```

Comment:
does libram not require you to specify the return type?

Reply:
I don't think it does, and I don't love doing it!

Reply:
per discussion in discord, my official hot take is "it should require you to do that but also we should find a way to make doing that not suck"